documentation/quarkus/getting-started-quarkus.asciidoc

```diff
@@ -0,0 +1,234 @@
+.. Migrate a Spring app to Quarkus + 
+This guide shows, how to link:../guide-migration-spring-quarkus.asciidoc[migrate a Spring application to a Quarkus application with devon4j conventions]. 
+
+
+== Create a REST service
+Now let's create our first REST CRUD service with Quarkus. 
+We give you the options to use a guide and start to code the service yourself,
+// TODO generate a service with CobiGen 
+or just download a service that's ready to use. 
+
+
+.Options
+. Create the service yourself +
+There is a good Quarkus guide for a simple https://quarkus.io/guides/rest-json#creating-your-first-json-rest-service[JSON REST service] that will guide you through your first application and will help you to implement defining endpoint with JAX-RS and an Entity that will be managed by the service and also how to https://quarkus.io/guides/rest-json#json[configure the JSON support].
+. Use an existing Quarkus project + 
+You don't want to code a service and just want to test some Quarkus functionalities? Just load a Quarkus https://github.com/quarkusio/quarkus-quickstarts#quick-start-list[sample project], provided for every existing quick start guide and the supported framework.
+Our Team also provides some Quarkus applications that are working and can be loaded and tested. 
+* https://github.com/devonfw-sample/devon4quarkus-reference[reference project] is a service that manages products. It contains the devon4j modern project structure, pagination, queries, a Postgres database, Jaeger tracing, Prometheus monitoring, SwaggerUI and support for Kubernetes deploy. 
+This project will be steadily improved and is used to showcase the abilities of Quarkus with devon4j.
+* https://github.com/devonfw-sample/devon4quarkus-minimal[minimal Quarkus project] is just the Quarkus project from a https://quarkus.io/guides/getting-started[getting started with Quarkus] guide with a **Greeting Services** modified with the correct modern structure talked about in this chapter <<Project Structure>>.
+
+
+== OpenAPI generation 
+
+We provide a guide with a short introduction to the link:../guide-openaoi.asciidoc[OpenAPI specification] with two plugins that are important in a Quarkus Context.
+
+1. link:../guide-openaoi.asciidoc#smallrye-openapi[Smallrye Openapi supported by quarkus] 
+2. link:../guide-openaoi.asciidoc#servicedocgen-maven-plugin[servicedocgen Maven Plugin].
+
+A more detailed usage guide to the Smallrye Plugin is provided by https://quarkus.io/guides/openapi-swaggerui[Quarkus OpenAPI and Swagger guide].
+
+
+== Testing a Quarkus Application 
+After we have built the service, we have to verify it with some tests.
+We will give you some guidelines to implement some test cases.
+
+.Testing Guides 
```

Comment:
I'm not sure, but wouldn't it make more sense to have the testing part after the database part?